packages/liveblocks-node/src/client.ts

```diff
@@ -130,7 +157,13 @@ type SchemaPlain = DateToString<Schema>;
 /**
  * Interact with the Liveblocks API from your Node.js backend.
  */
-export class Liveblocks {
+export class Liveblocks<
+  _P extends JsonObject = DP,
```

Comment:
@GuillaumeSalles @marcbouchenoire @nimeshnayaju I would like to hear your professional opinion here.

The most pragmatic approach here for now may be to remove the unused `P` param here. Or do we anticipate our Node client ever going to read or set Presence somehow? I don't think it makes much sense.

However, if we remove it here now, and it turns out later that we need to add it in anyway, then we must add it at the end and live with the inconsistent param order. (Not a big issue apart from the aestetics/consistency.)

I'm for removing it here now, and just having this client take 4 out of our 5 params instead. Thoughts?

Reply:
I would remove all the generic types on the `Liveblocks` class and only use module augmentation for the launch. Let's wait until we get enough users asking for this level of control. Generics are easy to add later on.

Reply:
Ah, yes. That might actually be nice. They are harder to remove than to add. Perfect. Goes without saying that in that case you can only use the client for a single room type. If your app requires multiple room types, it’s not going to be possible.

I’m on board with that until someone needs it 👍

Will remove them before meeting.

Reply:
@GuillaumeSalles While making the change, I realized that one downside is that unit tests [like this one](https://github.com/liveblocks/liveblocks/blob/736a08abbdc37362b77730aca28a57ac04b84ac1/packages/liveblocks-node/src/__tests__/client.test.ts#L581-L585) could no longer be used to test one-off edge cases. If the only way to control the types is the `global`, then all tests will always use the same types, and it's also impossible to test them for a user without augmentation 🤔

I think to get past this limitation, I'll export a private class that still has these type params, but bind them to `DS`, `DU`, etc in our public export. That way, users will not have the type params, but we can still use them in unit tests?

Reply:
Mhh, I don't think tests should influence external API and internal complexity that much.

I believe it's okay to adapt the tests by adding default metadata all the time or cast to `any` if this is the only choice. It just feels simpler to maintain and more pragmatic, no?

Reply:
A'ight, did that in 2bfa19799a573845bb9b3b23500fcc457bbaa2d8 now 👍  I think this is ready for merge.